secant/MockedDependencies/KeyStoring.swift

```diff
@@ -7,32 +7,45 @@
 
 import Foundation
 
+/// Representation of the wallet stored in the persistent storage (typically keychain, handled by `RecoveryPhraseStorage`).
+struct StoredWallet: Codable, Equatable {
+    var birthday: BlockHeight?
+    let seedPhrase: String
```

Comment:
Yes here's where the locale could be added.  Even if it is always the same value, having it here would make things easier because we don't have to worry about the `Codeable` serializers changing.

Reply:
Could we rely on?
```swift
    public static func determineLanguage(from mnemonicWords: [String]) throws -> MnemonicLanguageType {
        guard mnemonicWords.count > 0 else {
            throw MnemonicError.wrongWordCount
        }

        if String.englishMnemonics.contains(mnemonicWords[0]) {
            return .english
        } else if String.chineseMnemonics.contains(mnemonicWords[0]) {
            return .chinese
        } else {
            throw MnemonicError.unsupportedLanguage
        }
    }
```

It's already returning unsupportedLanguage. The method is clearly counting with uniqueness in the dictionaries but I think this would work for many possible languages as well. Extending it to spanish, french, german would work as well. 

@pacu @ccjernigan ?

Reply:
Otherwise if we really want to store `MnemonicLanguageType` we need to update the MnemonicSwift repo so the enum is `Codable`

Reply:
As we spoke we probably want to store the locale of the phrase but only accept english to future-proof the bundle and avoid a potential migration.